docs/nddata/ccddata.rst

```diff
@@ -123,21 +123,24 @@ more details about bit planes and the functions ``astropy`` provides for
 converting them to binary masks, see :ref:`bitmask_details`. For more details
 on setting flags, see `~astropy.nddata.NDData`.
 
-WCS
-+++
+WCS and PSF
```

Comment:
My preference would be to make PSF its own bullet point, partly because the way that WCS and PSF are populated is vary different -- WCS comes from the header and PS from an image extension.